Classes.lua

```diff
@@ -4209,6 +4215,37 @@ if Hekili.IsClassic() then
     } )
 end
 
+if Hekili.IsClassic() then
+    all:RegisterAura( "tol'vir_agility", {
+        id = 79633,
+        duration = 25,
+        max_stack = 1
+    })
+    all:RegisterAbility( "tol'vir_agility", {
+        id = 79633,
+        known = function () return tinker.hand.spell == 79633 end,
+        cast = 0,
+        cooldown = 1,
+        gcd = "off",
+
+        item = function() return tinker.hand.spell == 79633 and tinker.hand.item or 0 end,
+        itemKey = "tol'vir_agility",
+
+        texture = function() return tinker.hand.spell == 463923 and tinker.hand.texture or 0 end,
+        startsCombat = true,
+
+        toggle = "tol'vir_agility",
```

Comment:
This isn't an actual toggle that exists.

Reply:
wilö take it out